index.html

```diff
@@ -210,4 +210,4 @@ <h6>About me</h6>
       </section>
     </div>
   </body>
```

Comment:
- [ ] Great job adding the about section. Kindly add the background image in the about section to match the figma template. You are expected to stick to design as much as possible including the background images.
### your design
![image](https://user-images.githubusercontent.com/94551484/227712173-f541f333-7855-49b5-ac06-09ab84fc9aaa.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/227712208-d0d4f9c9-c490-40ed-aacc-0a1caf5e4b5d.png)